ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/ui/catalogueElementTreeviewItem.coffee

```diff
@@ -180,6 +188,20 @@ angular.module('mc.core.ui.catalogueElementTreeviewItem', ['mc.util.names', 'mc.
         $scope.collapseOrExpand()
         $scope.treeview.select(element)
 
+      $scope.metadataOccurrencesToAsterisk = (element) ->
+        # only Data Element with some metadata and defined relationship
+        if element.$$metadata?.values? && element.getElementTypeName() == 'Data Element' && element.$$relationship
```

Comment:
@musketyr ok so I resolved it by checking if there is any relationship for Data Element, if not, it is probably just a listing of data elements. May it be like that? :eyes: